desk/sur/activity.hoon

```diff
@@ -0,0 +1,57 @@
+|%
++$  stream=(mop time event)
++$  indices=(map concern stream)
++$  concern
+  $%  [%group group-concern]
+      [%channel channel-concern]
+      [%dm dm-concern]
+      [%post post-concern]
+      [%writ writ-concern]
+  ==
++$  group-concern    group=flag
++$  channel-concern  [channel=nest group=flag]
++$  dm-concern       =whom
++$  post-concern     [=post-key channel=nest group=flag]
++$  writ-concern     [=writ-key =whom]
```

Comment:
This feels like the following is being implied here:

```suggestion
+$  group-concern    group=flag
+$  channel-concern  [channel=nest group-concern]
+$  dm-concern       =whom
+$  post-concern     [=post-key channel-concern]
+$  writ-concern     [=writ-key dm-concern]
```

As in, these "scopes" nest a certain way. But channel events don't appear in the group event stream also, do they? You'd construct a "mixed" stream like that on-demand, instead of through the way it's structured/stored?